.github/workflows/update-db.yaml

```diff
@@ -0,0 +1,41 @@
+name: Update databases schema
+
+on:
+  push:
+    # Run workflow when the schema changes.
+    branches: [main]
+    paths:
+      - "data-serving/data-service/schemas/**"
+
+jobs:
+  update-case-data:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
```

Comment:
I remember Kelly having to install the MongoDB CLI to do this (and writing d.ts for it in the npm script). Can't recall if that was for this schema update portion, or something else, though. Might be good to test this works in the actions VM by temporarily committing a pull_request trigger.

Reply:
this was needed because she was running a different script that was doing a mongoimport, this script only deals with the schema and uses the node js mongo client so it should be fine (confidence lelvel 90%, merging this now ;) )